package.json

```diff
@@ -18,15 +18,26 @@
     "start:automated": "webpack-dev-server --color --progress --config webpack.test.js",
     "start": "webpack-dev-server --color --progress --config webpack.dev.js",
     "example": "ws --rewrite '/* -> /dist/$1' --https",
-    "prod": "npm run build",
+    "prod": "npm run build && npm run copy",
     "snyk": "snyk test --org=secure-trading",
     "test": "jest --watchAll",
     "coverage": "jest --coverage",
+    "copy": "copy-files-from-to",
```

Comment:
info: very useful plugin; it copies files `from-to` and we don't have to worry about complicated copy-paste syntax anymore ;)